src/pages/Home/index.tsx

```diff
@@ -37,11 +47,24 @@ const Home = () => {
           {vouchers?.data
             ? vouchers.data?.map((voucher) => (
                 <VoucherCard
+                  isRedeemable = {false}
+                  hasRedeemed = {true}
```

Comment:
Nitpick: Can you remove the spaces between `=` for component props. And also you don't need to add `={true}` for props that are `true`. `hasRedeemed` only would suffice